Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -67,6 +67,79 @@ struct TextureMapperLayer::ComputeTransformData {
     }
 };
 
+class TextureMapperFlattenedLayer final {
```

Comment:
I would prefer this to move into its own .cpp/h file -- this is already large enough.

Reply:
I'm planning to do some refactoring to unify tiled updates which are also used in other places in very similar manner. Maybe it's better to move this to its own file in separate patch if the class still grows in size. Now it's quite small and only for internal usage of TextureMapperLayer.

Reply:
Still, we have a general policy of one class per .cpp/h file in WebKit, which was already violated, and this doesn't make it better :-)

Reply:
OK, usually I tend to do this kind of refactoring before in a separate PR. But we can also postpone the cleanup to a follow-up PR. I don't want to block this important work.

Reply:
ok, I'll do follow up patch